README.md

```diff
@@ -113,244 +112,128 @@ bash test/test_cleanup.sh
-kept.
-
-#### _QUANTIFY_
-
-The third and final subworkflow, **_QUANTIFY_**, quantifies miRNA expression by
-intersecting the alignments from the **_MAP_** subworkflow with the annotations
-generated in the **_PREPARE_** subworkflow. Intersections are computed with
-[`bedtools`][bedtools] for one or multiple of mature, primary transcripts and
-isomiRs. Reads consistent with each miRNA are counted and tabulated.
-
-
-The schema below is a visual representation of
-an example run of the whole workflow:
-
-> ![rule-graph][rule-graph]
-
-
-### Running the workflow
-
 Assuming that you are currently inside the repository's root directory, 
-create a directory from which you will run your workflow and
-name it whatever you want e.g., `MY_ANALYSIS` and 
-head to it.
+create a directory from which you will run your workflow and name it whatever 
+you want e.g., `MY_ANALYSIS` and head to it.
 
 ```bash
 mkdir MY_ANALYSIS
 cd MY_ANALYSIS
 ```
-Now within the `MY_ANALYSIS/` directory, 
-create a new directory to store you library; 
-name it as you like, e.g. `my_lib` and move there.    
+Place your library file(s) here. In addition, create the sample table 
+`sample_table.csv` and fill it up. The `sample_table.csv` is a tab-separated  
+file, with one row per library, that must have the following columns:  
 
-```bash
-mkdir my_lib
-cd my_lib
-```
-Place your library file(s) here. 
-In addition, create a sample table. 
-Fill it with the correct entries. 
-You can look at the `test/test_files/sample_table.csv` 
-to get an idea of what this file must look like.
+- `sample`. This column contains the library name.  
+- `sample_file`. In this column, you must provide the path to the library file.
+The path must be relative to the working directory.  
+- `adapter`.  This field must contain the adapter sequence in capital letters.  
+- `format`. In this field you mast state the library format. It can either be 
+`fa` if providing a FASTA file or `fastq` if the library is a FASTQ file.  
+
+You can look at the `test/test_files/sample_table.csv` to know what this file 
+must look like, or use it as a template.
 
 ```bash
 touch sample_table.csv
 ```
+In this same directory, add the following required input files:  
+
+- The gzipped reference genome in FASTA format.  
+- The gzipped gene annotation file in GTF format.  
+- The unzipped microRNA annotation file in GTF format.  
+- A tab-separated mappings table between UCSC and Ensembl.
 
-Now traverse back to the previous directory.
-Here, a couple of things must be done:  
+> **Note:** We expect the genome and gene annotations to be formatted according
+> the style used by Ensembl. Other formats are very likely to lead to problems.
+> The miRNA annotation file is expected to originate from miRBase, or follow 
+> their exact layout.
 
-First of all, copy the `config_template.yaml` to this directory.
+Finally, copy the `config_template.yaml` to this directory.
```

Comment:
Again, put a separate subsection "Prepare configuration file". And again, it does not need to be in the same directory, it could be anywhere.